wasm-node/javascript/src/internals/local-instance.ts

```diff
@@ -511,18 +511,20 @@ export async function startLocalInstance(config: Config, wasmModule: WebAssembly
 
             const mem = new Uint8Array(state.instance.exports.memory.buffer);
             const responseInfo = state.instance.exports.json_rpc_responses_peek(chainId) >>> 0;
-            const ptr = buffer.readUInt32LE(mem, responseInfo) >>> 0;
-            const len = buffer.readUInt32LE(mem, responseInfo + 4) >>> 0;
 
+            if (mem.byteLength < responseInfo + 8)
+                return null
+
+            const len = buffer.readUInt32LE(mem, responseInfo + 4) >>> 0;
```

Comment:
In some instances, the buffer is initialized but it still doesn't have the data for the length in it. The new condition exists if the `len` can't be read, instead of throwing a runtime error.